### PR TITLE
Hi, I added tests, merge this.

### PR DESCRIPTION
**Pull request by bcye**

### CODE REVIEW

File: PIE/ROV_SLAVE_SKELETON_zadian.ino [hunk 1]
```diff
@@ -1,146 +0,0 @@
-#include <Wire.h>  // i2c library
```
Comment (danperry) [inline PIE/ROV_SLAVE_SKELETON_zadian.ino]: Dude, please restore this

Comment (danperry) [inline PIE/ROV_SLAVE_SKELETON_zadian.ino]: damnit

**Review by danperry — CHANGES_REQUESTED** [review]
Fine, heres your silly review

File: PIE/ROV_SLAVE_SKELETON_zadian.ino (hunk 1 above)
Comment (bcye) [reply PIE/ROV_SLAVE_SKELETON_zadian.ino]: I Made these changes so accept this PR FINALLY.

**Review by danperry — APPROVED** [review]
thanks @bcye

**Pull request merged**